### PR TITLE
 Implement seed script to write images, reviews, and subRatings to CSV

### PR DESCRIPTION
**Pull request by lydiahu8**
1) Add 3 functions to generate data for images table, reviews table, and subRatings table
2) Add a function for writing data to csv file

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,7 +3,12 @@ node_modules
 package-lock.json
 .env
 client/dist/bundle.js
-
+database/helpers/reviews.csv
```
Comment (MichaelHamm) [inline .gitignore:6]: if you want you can add *.csv so that all csvs are ignored

File: database/seed_pgs.js [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+  const reviewId = randomNumberGenerator(0, 2e7);
+
+  return `'${imageUrl}',${reviewId}\n`;
+};
+
+// Generates each row for the subRatings table
+const generateSubRatings = () => {
+  const feature = features[randomNumberGenerator(0, 8)];
+  const rating = randomNumberGenerator(1, 5);
+  const reviewId = randomNumberGenerator(0, 2e7);
```
Comment (MichaelHamm) [inline database/seed_pgs.js:123]: What is 2e7?

**Review by MichaelHamm — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (lydiahu8) [reply .gitignore:6]: awesome! I added it to the .gitignore file

File: database/seed_pgs.js (hunk 2 above)
Comment (lydiahu8) [reply database/seed_pgs.js:123]: 2e7 = 20,000,000

**Pull request merged**